pkgs/tools/security/gopass/default.nix

```diff
@@ -1,4 +1,11 @@
-{ stdenv, buildGoModule, fetchFromGitHub, git, gnupg, xclip, wl-clipboard, installShellFiles, makeWrapper }:
+{ stdenv, makeWrapper
+, buildGoPackage, fetchFromGitHub, installShellFiles
```

Comment:
You changed this here to `buildGoPackage` but further down you are using `buildGoModule`

Reply:
Oops, yes, that seems like a rebase artifact - the package was using `buildGoPackage` before

Reply:
Fixed it (and built locally as well this time :P)